README.md

```
@@ -1,199 +1,56 @@
-# Dev-Prep — 실전형 타이핑 면접 연습 서비스 (Frontend)
+## Dev-Prep — 타이핑 기반 면접 연습
 
-타이핑 기반 면접 연습을 **실제 면접 환경처럼** 경험할 수 있는 React + TypeScript 기반 웹 애플리케이션입니다.  
-GitHub OAuth 인증, 제한 시간, 자동 제출, 질문 세트 관리 등 실전 면접 흐름을 웹 상에서 재현하는 것을 목표로 개발했습니다.
+면접 질문을 타이핑으로 답하고, 제한시간 내 제출하는 연습 환경을 제공합니다. <br/> 현재 AI 피드백/모범 답안은 개발 예정이며, 관련 UI는 구현되어 있습니다.
 
----
+#### 핵심 기능
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)